src/App.tsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+type State = {
+  text: string;
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    text: '',
+  };
+
+  componentDidMount(): void {
+    this.handleState();
+  }
+
+  componentDidUpdate(): void {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
```

Comment:
[checklist] use componentDidMount to add keyup handler. 

You added event listener for componentDidUpdate, but you don't need to use componentDidUpdate in this situation.